### PR TITLE
Moves Files adapters to external packages

### PR DESCRIPTION
**Pull request by flovilmart**

### CODE REVIEW

File: src/Adapters/Files/FilesAdapter.js [hunk 1]
```diff
@@ -13,22 +13,22 @@
 
 export class FilesAdapter {
   /* this method is responsible to store the file in order to be retrived later by it's file name
-   * 
-   * 
+   *
+   *
    * @param config the current config
```
Comment (drew-gross) [inline src/Adapters/Files/FilesAdapter.js]: Update comments

**Comment by drew-gross** [comment]
Looks awesome to me but I guess because of removal of `config` parameter it's a breaking change for any other file adapters (are there any?) so I would like a second opinion @gfosco @nlutsenko

**Comment by nlutsenko** [comment]
It definitely is, but looks like a good change. We would need to increment a minor version for the next release, hence breaking changes. I wish we moved GridStoreAdapter as well...

**Comment by drew-gross** [comment]
Lets add some info about the breaking change and how to deal with it in the changelog or release notes or something, then merge.

**Pull request merged**